spec/vector_spec.rb

```diff
@@ -502,11 +502,32 @@
             expect(sorted).to eq(Daru::Vector.new(["My", "Jazz", "Guitar", "My Jazz Guitar"], index: [2,1,3,0]))
           end
 
-          it "places nils near the beginning of the vector" do
+          it "places nils near the beginning of the vector when sorting ascendingly" do
             with_nils = Daru::Vector.new [22,4,nil,111,nil,2]
 
             expect(with_nils.sort).to eq(Daru::Vector.new([nil,nil,2,4,22,111], index: [2,4,5,1,0,3]))
           end if dtype == :array
+
```

Comment:
I had placed that there because NMatrix 0.1.0 would segfault with nils sometimes. I guess that does not happen anymore with NMatrix 0.2.1. So no need.